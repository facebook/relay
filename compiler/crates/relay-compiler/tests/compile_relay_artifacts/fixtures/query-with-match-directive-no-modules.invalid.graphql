# expected-to-throw
query queryWithMatchDirectiveNoModules_NameRendererQuery($id: ID!) {
  node(id: $id) {
    ...queryWithMatchDirectiveNoModules_NameRendererFragment
  }
}

fragment queryWithMatchDirectiveNoModules_NameRendererFragment on User {
  id
  nameRenderer @match {
    __typename
  }
}
