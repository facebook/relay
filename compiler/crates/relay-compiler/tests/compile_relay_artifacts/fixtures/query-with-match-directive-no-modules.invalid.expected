==================================== INPUT ====================================
# expected-to-throw
query queryWithMatchDirectiveNoModules_NameRendererQuery($id: ID!) {
  node(id: $id) {
    ...queryWithMatchDirectiveNoModules_NameRendererFragment
  }
}

fragment queryWithMatchDirectiveNoModules_NameRendererFragment on User {
  id
  nameRenderer @match {
    __typename
  }
}
==================================== ERROR ====================================
✖︎ Invalid @match selection: expected at least one @module selection. Remove @match or add a '...Fragment @module()' selection.

  query-with-match-directive-no-modules.invalid.graphql:10:16
    9 │   id
   10 │   nameRenderer @match {
      │                ^^^^^^
   11 │     __typename
